Tutorials/start/configuring-proxy.md

```diff
@@ -1,124 +1,42 @@
 
-An incorrect COM/serial port assignment or unconnected Faraday radio will cause the following common error to appear.
+*prox.ini* comes mostly configured so little needs to be changed. However, we do require changes to `--callsign`, `--nodeid`, and `--port` options.
+* `--callsign CALLSIGN` replace CALLSIGN with the Faraday radio callsign
+* `--nodeid NODEID` replace NODEID with an integer between 0-255 to ID the radio
+* `--port PORT` replace PORT with the UART port faraday is connected to such as *COM10*
 
-![Proxy Connection ERROR](images/Proxy-Error-Connection.png "Proxy Connection ERROR")
+The entire command will look something like this:
+`faraday-proxy --callsign kb1lqc --nodeid 2 --port COM23`
 
-> Check your COM port settings and that you did not change `baudrate` in `proxy.ini`
+![Proxy Configuration Example](images/proxy-configuration.jpg "Proxy configuration")
 
-## Connecting To Multiple Faraday Devices
+Proxy will automatically start when this command is run. Once configured you do not have to reconfigured unless desired and can start Proxy with `faraday-proxy`.
 
-> This feature is currently broken in `Master` and we are actively working on [Issue #86](https://github.com/FaradayRF/Faraday-Software/issues/86) to fix it!
+> * Linux users may have a port value such as `--port /dev/ttyUSB0`
+> * Max OS X users may have a port value such as `--port /dev/cu.usbserial-40`
 
-The proxy interface can connect to more than one Faraday digital radio at a time and this is achieved by creating more instances of `[UNITx]` sections and updating the `UNITS` value in the `[PROXY]` section.
+### Proxy Running
+Proxy provides a server which other programs can communicate with the radio through and therefore must always be run when using the hardware. We suggest giving and entire command prompt window to Faraday as it tends to print out information when queried.
 
-### Windows
- 1. [Connect](connecting-hardware.md) ***both*** Faraday radios to your computer via USB and identify their respective COM ports. 
- 2. Open and configure the `proxy.ini` by changing `UNITS` in the [PROXY] section to indicate the quantity of radios connected to the computer. Also add a `[UNITx]` section for each radio.
+![Proxy INI](images/proxy.jpg "Proxy command prompt")
+
+Congratulations, Proxy is now running successfully!
 
-A proper ```proxy.ini``` configuration file will resemble the example below.
+#### UARTStack Fail Messages
+Do not be concerned if you see any messges such as that shown below. This is simple a bad packet sent to Proxy over UART which failed CRC checking and was thrown out. This occasionally happens and is OK. With noisy computer grounds and poor quality USB cables one can expect to see more of these messages.
```

Comment:
Action @kb1lqc : Typo "This is simple a bad..." should be "This is simply a bad..."